CMake/DefineOptions.cmake

```diff
@@ -28,19 +25,24 @@ option(WITH_GPL_LIBS "Build with GPL libraries." ON)
 
 if(WIN32)
   option(WITH_MINIMAID "Build with Mimimaid Lights Support." OFF)
-else()
-  if(LINUX)
+elseif(LINUX)
     option(WITH_FFMPEG "Build with FFMPEG." ON)
     # Builder beware: later versions of ffmpeg may break!
     option(WITH_SYSTEM_FFMPEG "Build with the system's FFMPEG." OFF)
     option(WITH_CRYSTALHD_DISABLED "Build FFMPEG without Crystal HD support." OFF)
     option(WITH_TTY "Build with Linux TTY Input Support." OFF)
     option(WITH_PROFILING "Build with Profiling Support." OFF)
-    option(WITH_GLES2 "Build with OpenGL ES 2.0 Support." ON)
+    #option(WITH_GLES2 "Build with OpenGL ES 2.0 Support." OFF)
     option(WITH_GTK2 "Build with GTK2 Support." ON)
     option(WITH_OGG "Build with OGG/Vorbis Support." ON)
     option(WITH_MP3 "Build with MP3 Support." ON)
     option(WITH_PARALLEL_PORT "Build with Parallel Lights I/O Support." OFF)
     option(WITH_CRASH_HANDLER "Build with Crash Handler Support." ON)
-  endif()
 endif()
+
+if(NOT ANDROID)
+  # This option handles if we use SSE2 processing.
+  option(WITH_SSE2 "Build with SSE2 Optimizations." ON)
+elseif(ANDROID)
+  option(WITH_CRASH_HANDLER "Build with Crash Handler Support." ON)
```

Comment:
Considering that Linux was working just fine without the crash handler (except when it crashes), should this perhaps be an option for all but Windows instead?